stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardscan/CardScanActivity.kt

```diff
@@ -0,0 +1,683 @@
+package com.stripe.android.stripecardscan.cardscan
```

Comment:
It seems like there are a lot of opportunities to share code between this activity and the CardImageVerificationActivity. Can we split some of this functionality out into a super class or interface that both this and `CardImageVerificationActivity` extend from?